scripts/check_samples.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Google LLC
```

Comment:
Please check this against the Google Python style guide: http://google.github.io/styleguide/pyguide.html

Reply:
not too sure if I got everything right, but I updated few things that I saw were wrong and my pylint3 is now good (just one line >100 chars)